docs/source/installation/win.rst

```diff
@@ -1,2 +1,95 @@
 Windows
 =======
+
+Cairo installation
+******************
+
+1. Visit the official `Download Page
+   <https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo>`_.
+
+2. Select the download that corresponds to your PC's architechture and Python
+   Version
+
+   .. image:: ../_static/installation/windows_cairo.png
+       :align: center
+       :width: 400px
+       :alt: windows cairo download page
+
+   .. note:: ``cp38`` corresponds to Python 3.8, ``cp37`` corresponds to Python
+             3.7, etc. ``win_amd64`` corresponds to 64-bit machines, ``win32``
+             corresponds to 32-bit machines.
+
+3. Open up your command prompt by hitting the Win key and typing ``cmd``.
+
+4. Enter the directory where you want to install cairo by typing ``cd
+   C:\path\to\cairo`` with the path being where you downloaded the ``.whl``
+   file in step 2.
+
+5. Finally, run ``pip3 install (cairo-file).whl``, where the file the
```

Comment:
Pip3 is not a valid command for windows so it should just `pip` or `pip3.7` with pip along with python version.